controllers/gslb_controller.go

```diff
@@ -265,10 +264,12 @@ func (r *GslbReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			},
 		}
 
-		strToInt := func(str string) int {
-			intValue, err := strconv.Atoi(str)
+		strToInt := func(s string) int {
+			intValue, err := strconv.Atoi(s)
 			if err != nil {
-				log.Err(err).Msg("can't convert string to int")
+				log.Err(err).
+					Str("value", s).
+					Msg("Can't convert string to int")
 			}
 			return intValue
 		}
```

Comment:
I'm checking the code and see following:
```go
		strToInt := func(str string) int {
			intValue, err := strconv.Atoi(str)
			if err != nil {
				log.Err(err).Msg("can't convert string to int")
			}
			return intValue
		}

		for annotationKey, annotationValue := range a.GetAnnotations() {
			switch annotationKey {
			case dnsTTLSecondsAnnotation:
				gslb.Spec.Strategy.DNSTtlSeconds = strToInt(annotationValue)
			case splitBrainThresholdSecondsAnnotation:
				gslb.Spec.Strategy.SplitBrainThresholdSeconds = strToInt(annotationValue)
			}
		}

```

`strToInt` is private embeded function parsing value from annotation. Thanks to the fact that the function is not used anywhere else and has one purpose (annotation parsing), we could better target the error message. Instead of the generic "Can't convert string" we could say something like, "I can't parse the annotation... " or something like that.

Reply:
Done